### DISCUSSION TITLE
Is there an option to skip "release" process while deploying

### DISCUSSION THREAD

**Discussion by dirkzado** [dokku/dokku#7249] [Q&A]
Sometimes I need to skip the "release" process (in Procfile). Is there such an option like `dokku git:from-image APP docker_repository:app-branch *SKIP=release*` ?

**Comment by josegonzalez** [comment]
No there is no such option at this time. Why do you need to do that?

**Reply by dirkzado** [reply]
My idea was that when you have two or more application server you might run the db migrations only on one of the application server. And there is a haproxy in front of it.

I know that the kubernetes version exists but with the limitation of not using nginx and multiple domain certification (and wildcard sub-domain cert).

Since I really like dokku I try to find some solutions for my issues ;)

But I think I have found an workaround for my special case: in Procfile and the release Iine I call an shell script which checks an environment variable and then skips a db migration.

**Reply by josegonzalez** [reply]
The kubernetes integration (that uses k3s underneath) supports either traefik or nginx as load balancing systems. While we haven't exposed it, it _should_ be possible to use a custom wildcard cert (how that gets provisioned kinda depends on your needs tbh though).

**Reply by dirkzado** [reply]
Thank you for your answer! I tried k3s kubernetes integration some months ago and had this issue:  https://github.com/dokku/dokku/discussions/6962
Is it worth a new try?

**Reply by josegonzalez** [reply]
Mind filing a ticket for wildcard certificates?

**Reply by dirkzado** [reply]
https://github.com/dokku/dokku/issues/7257#issue-2575745551
